callmetadata.schema.json

```diff
@@ -5,31 +5,41 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
+        "member": {
+            "name": {
+                "type": "string",
+                "example": "Steve Jones",
+                "description": "Name of the participant communicating as defined in schema.org"
```

Comment:
The statement "as defined in schema.org" does not make sense in this context, right? I suggest keep only "Name of the participant in the communicating"